loopchain/channel/channel_inner_service.py

```diff
@@ -613,7 +614,7 @@ def get_tx_info(self, tx_hash):
             return message_code.Response.success, tx_info
         else:
             try:
-                return message_code.Response.success, self._block_manager.get_tx_info(tx_hash)
+                return message_code.Response.success, self._blockchain.find_tx_info(tx_hash)
             except KeyError as e:
                 logging.error(f"get_tx_info error : tx_hash({tx_hash}) not found error({e})")
```

Comment:
It should be better if the below log is updated with the edited method name.

Reply:
Please check that the log line is changed as you intended.